src/components/ec-dropdown-search/ec-dropdown-search.vue

```diff
@@ -334,24 +365,45 @@ function isItemSelected(item) {
 const selectedItemIndex = computed(() => filteredItems.value.indexOf(toRaw(props.modelValue)));
 
 // CTA
+const itemsOverflowContainer = ref(null);
 const isCtaAreaFocused = ref(false);
+const { deactivate, activate } = useFocusTrap(itemsOverflowContainer, getFocusTrapOptions());
+function getFocusTrapOptions() {
+  return {
+    immediate: false,
+    escapeDeactivates: true,
+    clickOutsideDeactivates: true,
+    fallbackFocus: popoverWrapper,
```

Comment:
This fallbackFocus option do not change too during the runtime. So there is no reason to create function.

`fallbackFocus` should be `HTMLElement` or `SVGElement` or `string` or `Function`, but `popoverWrapper` is a `Ref`. 

I suppose we should pass here `() => popoverWrapper.value` as a fallbackFocus.

Reply:
corrected, thanks